config/locales/en.yml

```diff
@@ -179,6 +179,11 @@ en:
       route_deleted: Question %{question_position}’s route has been deleted
       route_updated: Question %{question_position}’s route has been updated
       title: Success
+  bulk_options:
+    hint: List your options, with each option on a new line.
+    label: Add options to your list
+    longer_than_30_options: If your list is longer than 30 options people will be able to search and find the options by typing in an autocomplete box.
+    select_one_option: People will be able to select one option from a list.
```

Comment:
I went with "your" list on the other page - the designs were unclear which it should be, but this can be picked up in the content sweep.

I wonder whether we should centralise the shared content in one place, but we can consider that later.

Reply:
I've updated to 'your list' for consistency, let's see what the content sweep brings